src/main/java/seedu/address/logic/Logic.java

```diff
@@ -65,4 +67,10 @@
      * @return Path of contact list.
      */
     public Path getContactListFilePath();
+
+    public ReadOnlyEventList getEventList();
+
+    public ObservableList<Event> getFilteredEventList();
+
+    public Path getEventListFilePath();
```

Comment:
The other methods in this interface do not mention the access modifier, should we standardise?

Reply:
I will update this later on!